docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,182 @@
+If deployed on kubernetes, consider running the app on a StatefulSet workload, for persistent storage.
+
+After the server has been provisioned, a database must be created with proper credentials (username and password), required later when starting the `Airy apps`.
+
+We have provided a helm chart, which can deploy PostgreSQL server in kubernetes, after setting the proper kubernetes config file and context. For creating the PostgreSQL server with helm, set the `pgPassword` variable in `infrastructure/helm-chart/charts/postgres/values.yaml` file and run:
+```sh
+helm install postgres infrastructure/helm-chart/charts/postgres/
+```
+
+### Redis
+
+A cluster of redis servers can be used or a managed one such as AWS ElastiCache for Redis
+If deployed on kubernetes, consider running the app on a StatefulSet workload, for persistent storage.
+
+We have provided a helm chart, which can deploy the Redis server in kubernetes, after setting the proper kubernetes config file and context. For creating the redis cluster with helm run:
+```sh
+helm install redis infrastructure/helm-chart/charts/redis/
+```
+
+### Kubernetes or docker runtime
+
+The `Airy apps` run on top of kubernetes, which is our preferred runtime. All of the apps can also be deployed using docker, however we recommend kubernetes for more robust and reliable deployment in production environments.
+
+In order to proceed with deploying the apps, we assume that you have a running kubernetes cluster, properly configured KUBECONF file and properly set context.
+
+At the moment, the Airy Core Platform is fully tested on K3s, Minikube and AWS EKS. All of the following configuration and deployment instructions are applicable for all kubernetes implementations in general.
+
+### Configuration
+
+After the underlying services are deployed, the Airy apps need to be created as docker containers on any docker runtime or inside kubernetes. Regardless of the deployment decision, they must have network access to the underlying services through the following variables which need to be defined before starting the containers.
+
+Connecting the `Airy apps` to the Kafka cluster, PostgreSQL and Redis can be done by creating a configuration file, prior to deploying the apps.
+
+Example with all possible configuration items, can be found in `infrastructure/airy.conf.all`. This file should be copied to `airy.conf` and edited according to the particular environment.
+
+
+### Deployment of the Airy apps
+
+The Airy apps can run on a docker runtime engine, however we recommend deploying the Airy Core Platform inside Kubernetes. For bootstrapping the entire Airy Core Platform on a single machine, we included a Vagrant configuration, inside the `infrastructure` directory, which creates a K3s Kubernetes implementation, running on top of Alpine Linux. All of the underlying services and the Airy apps run inside this Kubernetes box, however, they can also be deployed in a different manner.
```

Comment:
this seems relevant to the local doc but now this one right?

Reply:
Will reword and keep a short paragraph. Will also put some of this content in the local doc.